.azure_pipelines/performance.yaml

```diff
@@ -0,0 +1,43 @@
+trigger: none
```

Comment:
BTW, since this pipeline takes time to run and we only plan to run this once a week.
should we turn of the pr trigger too? Like here? https://github.com/microsoft/Olive/blob/main/.azure_pipelines/build-doc.yaml#L12

Reply:
Or add some branch and path filters to only trigger this PR when the relevant files in the `.azure_pipelines` subdir are updated by a PR